src/marqo/core/index_management/index_management.py

```diff
@@ -86,15 +86,24 @@ def bootstrap_vespa(self) -> bool:
         Returns:
             True if Vespa was bootstrapped, False if it was already up-to-date
         """
-        with self._vespa_deployment_lock():
-            vespa_app = self._get_vespa_application(check_configured=False, need_binary_file_support=True)
 
-            to_version = version.get_version()
-            from_version = vespa_app.get_marqo_config().version if vespa_app.is_configured else None
+        # We skip the Vespa convergence check here so that Marqo instance can be bootstrapped even when Vespa is
+        # not converged. This will reduce the risk of having downtime when the Marqo instance crashes while waiting
```

Comment:
Description is confusing. Do you mean crashes when a deployment triggered by Marqo is converging? I think we're worried more about external deployments converging when Marqo has no changes to make and can start anyway

Reply:
> when a deployment triggered by Marqo is converging? 
I does not matter who triggers the deployment, it can be Marqo or other processes.